requirements.txt

```diff
@@ -32,4 +32,7 @@ timeseriesflattener==1.8.0
 wandb==0.15.4
 wasabi==0.9.1
 xgboost==1.7.3
-
+### awaiting approval from Andreas and Søren
+# synthcity==0.2.9 # currently forces a downgrade torch to 1.3.1..
+# imbalanced-learn==0.11.0
+# cleanlab==2.5.0
```

Comment:
If you remove these comments, we can see whether they induce new vulnerabilities :-)